build.sh

```diff
@@ -222,7 +222,7 @@ case $__HostOS in
         __monoroot=/usr/local
         ;;
     OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
+        __monoroot=/usr/local
```

Comment:
Ah okay cool; we should definitely have a mono path override. I'm touching the build.sh file in another pack so I'll look into adding that